src/chat/zulip/triagebot.md

```diff
@@ -0,0 +1,220 @@
+
+You can drop your claim to the issue via `@rustbot release-assignment`; Rust team members can do the same if they want to release someone else's assignment.
+
+`@rustbot assign @user` can be used only by Rust team members and will assign that user to the issue (with same rules as before -- either directly or indirectly).
+
+Soon (when the "highfive" bot migration will be complete, see [rust-lang/highfive#258](https://github.com/rust-lang/highfive/pull/258)), `r?` will also assign users to PRs, though unlike issues, non-team members cannot be assigned. Anyone can invoke the command.
+
+To enable on a repository, add the following to a triagebot.toml file in the repository root.
+
+```toml
+[assign]
+
+```
+
+## Issue notifications
+
+Each registered team member has a notifications page at:
+
+`https://triagebot.infra.rust-lang.org/notifications?user=<gh-username>`
+
+This page is populated from direct mentions (@user) and team mentions (@rust-lang/libs) across the rust-lang organization.
+
+It can also be edited via Zulip by [private-messaging triagebot](https://rust-lang.zulipchat.com/#narrow/pm-with/261224-triage-rust-lang-bot). Any Rust organization member can edit their notifications page, or pages of other Rust organization team members. To do so, the editor must have a `zulip-id` listed in their people/username.toml file in the [team repository](https://github.com/rust-lang/team/). The bot will tell you which ID to use when talking to it for the first time; please `r? @Mark-Simulacrum` on PRs adding Zulip IDs.
+
+The following commands are supported:
+
+ * `acknowledge <url>`
+ * `acknowledge <idx>`
+
+These both acknowledge (and remove) a notification from the list.
+
+ * `add <url> <description... (multiple words)>`
+
+This adds a new notification to the list.
+
+ * `move <from> <to>`
+
+This moves the notification at index `from` to the index `to`.
+
+ * `meta <idx> <metadata...>`
+
+This adds some text as a sub-bullet to the notification at `idx`. If the metadata is empty, the text is removed.
+
+ * `as <github username> <command...>`
+
+This executes any of the above commands as if you were the other GH user.
+
+## Ping a team
+
+The bot can be used to "ping" teams of people that do not have corresponding Github teams. This is useful because sometimes we want to keep groups of people that we can notify but we don't want to add all the members in those groups to the Github org, as that would imply that they are members of the Rust team (for example, Github would decorate their names with "member" and so forth). The compiler team uses this feature for their [ICE-breaker teams](https://rust-lang.github.io/rustc-guide/ice-breaker/about.html).
+
+TODO: isn't ICE-breakers now called [notification groups](https://rustc-dev-guide.rust-lang.org/notification-groups/about.html)?
```

Comment:
I saw this note [here](https://blog.rust-lang.org/inside-rust/2020/02/06/Cleanup-Crew-ICE-breakers.html#update), should ICE-breakers be removed as term?

Reply:
Yes, I think we've been moving away from the term "ICE-breakers". See also the PRs that have been merged to the rustc-dev-guide. This also means that the old link is broken, please use the new link as you suggested.